extensions/hp-extension/src/hp/hpMN.ts

```diff
@@ -19,9 +19,88 @@ export default {
   ],
   toolGroupIds: ['default'],
   stages: [
+    // {
```

Comment:
can you add at least a comment explaining why we commented it out?

Reply:
I uncommented it out - it was causing problems, but I figured out how to fix them.  Kind of an ugly fix, but will talk about it with Alireza later.